packages/stripe-integration/src/stripev3/StripePaymentMethod.spec.tsx

```diff
@@ -1,59 +1,37 @@
+/* eslint @typescript-eslint/no-floating-promises: 0 */
```

Comment:
we should migrate old `enzyme` test to `@testing-library/react`. But you can do this in separate PR. Create please a task for this, and for Stripe UPE as well.

Reply:
If you don't have problems with deadlines, I would suggest to implement it in this PR.

Reply:
No problem, I can do it in this PR

Reply:
@vitalii-koshovyi If you don't mind, I'll create a separate task for this.